src/cpp/include/openvino/genai/llm_pipeline.hpp

```diff
@@ -303,5 +303,12 @@ inline std::pair<std::string, Any> draft_model(
 */
 static constexpr ov::Property<SchedulerConfig> scheduler_config{"scheduler_config"};
 
+/**
+* @brief enable_prompt_lookup property serves to activate prompt lookup decoding.
+* Set `true` to activate this mode.
+* And create LLMPipeline instance with this config.
+*/
+OPENVINO_GENAI_EXPORTS std::pair<std::string, size_t> prompt_lookup(size_t max_ngram_size = 3);
```

Comment:
maybe it should be typical property like
```cpp
static constexpr ov::Property<size_t> max_ngram_size{"max_ngram_size"};
```
or we expect more parameters, required for `prompt_lookup`? 

For speculative decoding we introduced dedicated function as we need to combine multiple parameters to a single configuration property.

Reply:
BTW, maybe we need to add to `GenerationConfig`?

Reply:
Agree to extend genration_config